### PR TITLE
CKEditor external plugins not displayed

### PR DESCRIPTION
**Pull request by rdenarie**
Prior to this fix, CKEDITOR external plugins like insertPortalLink or insertLink were not displayed
The configuration in eXoConfig.js was erased by the javascript in UIFormRichTextInput
This fix remove the line about extraPlugins to let the configuration loads from eXoConfig.js file

The second part remove the extraPlugin helpBBCode
This plugin comes from forum, which is not included in meeds, and it generate error at startup.
As this plugin was present in 6.0.x due to current problem, I remove it from the configuration

### CODE REVIEW

File: commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java [hunk 1]
```diff
@@ -210,7 +210,6 @@ private String buildEditorLayout(WebuiRequestContext context) throws Exception {
             .append("CKEDITOR.plugins.addExternal('confirmBeforeReload','/commons-extension/eXoPlugins/confirmBeforeReload/','plugin.js');")
             .append("$('[name=\\'").append(name).append("\\']').ckeditor({")
             .append("customConfig: '/commons-extension/eXoConfig.js',")
-            .append("extraPlugins: 'confirmBeforeReload',")
```
Comment (ahamdi) [inline commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java:213]: This plugin was introduced to show a popup when a user adds an input in TextArea and tries to change/reload the page in this commit https://github.com/exoplatform/commons/commit/320df4870aaded70c607ab3f124412f6579737cb .
Could you add it by default in eXoConfig.js

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java (hunk 1 above)
Comment (rdenarie) [reply commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java:213]: done

File: commons-extension-webapp/src/main/webapp/eXoConfig.js [hunk 2]
```diff
@@ -20,9 +20,8 @@ CKEDITOR.editorConfig = function( config ){
   CKEDITOR.plugins.addExternal('simpleLink','/commons-extension/eXoPlugins/simpleLink/','plugin.js');
   CKEDITOR.plugins.addExternal('acceptInline','/eXoWCMResources/eXoPlugins/acceptInline/','plugin.js');
   CKEDITOR.plugins.addExternal('cancelInline','/eXoWCMResources/eXoPlugins/cancelInline/','plugin.js');
-  CKEDITOR.plugins.addExternal('helpBBCode','/forum/eXoPlugins/helpBBCode/','plugin.js');
```
Comment (ahamdi) [inline commons-extension-webapp/src/main/webapp/eXoConfig.js:23]: We need to add this line here 
`
CKEDITOR.plugins.addExternal('confirmBeforeReload','/commons-extension/eXoPlugins/confirmBeforeReload/','plugin.js');
`

File: commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java [hunk 3]
```diff
@@ -210,7 +210,6 @@ private String buildEditorLayout(WebuiRequestContext context) throws Exception {
             .append("CKEDITOR.plugins.addExternal('confirmBeforeReload','/commons-extension/eXoPlugins/confirmBeforeReload/','plugin.js');")
```
Comment (ahamdi) [inline commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java]: This line should be moved to eXoConfig.js to make the plugin available.

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java (hunk 3 above)
Comment (rdenarie) [reply commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java]: done

File: commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java (hunk 1 above)
Comment (rdenarie) [reply commons-webui-component/src/main/java/org/exoplatform/webui/form/UIFormRichtextInput.java:213]: done

File: commons-extension-webapp/src/main/webapp/eXoConfig.js (hunk 2 above)
Comment (rdenarie) [reply commons-extension-webapp/src/main/webapp/eXoConfig.js:23]: done

**Review by ahamdi — DISMISSED** [review]
LGTM

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**